mac

```diff
@@ -130,6 +130,8 @@ if ! brew_is_installed "node"; then
     fancy_echo 'Installing nvm and lts Node.js and npm...'
     touch ~/.bash_profile
     curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.7/install.sh | bash
+    export NVM_DIR="$HOME/.nvm"
+    [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh"
     nvm install --lts
```

Comment:
This will most likely fail without `nvm` being installed first. This is assuming that the `curl` command and `install.sh` script it runs install `nvm` correctly.